projectPlanning/issueTrackers/text.md

```diff
@@ -6,7 +6,7 @@
 
 <div id="body">
 
-Keeping track of project tasks %%(who is doing what, which tasks are ongoing, which tasks are done etc.)%% is an essential part of project management. In small projects it may be possible to track tasks using simple tools as online spreadsheets or general-purpose/light-weight tasks tracking tools such as Trello. Bigger projects need more sophisticated task tracking tools. 
+Keeping track of project tasks %%(who is doing what, which tasks are ongoing, which tasks are done etc.)%% is an essential part of project management. In small projects, it may be possible to track of tasks using simple tools such as online spreadsheets or general-purpose/light-weight task tracking tools such as Trello. Bigger projects need more sophisticated task tracking tools.
```

Comment:
```suggestion
Keeping track of project tasks %%(who is doing what, which tasks are ongoing, which tasks are done etc.)%% is an essential part of project management. In small projects, it may be possible to keep track of tasks using simple tools such as online spreadsheets or general-purpose/light-weight task tracking tools such as Trello. Bigger projects need more sophisticated task tracking tools.
```